foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -279,6 +296,187 @@ public class EdgeEventHandle {
 ```
 
 Use this class when calling the sendEvent API with EdgeCallback.
+
+#### ExperienceEvent
```

Comment:
Per my previous comment, let's break this out of the single tab group  and create an ExperienceEvent section with its own tab group. Do the same for the other classes. It will look similar to the Public Classes section of the Identity for Edge Network API reference. 
https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#public-classes